api/defaults/defaults.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package defaults defines Teleport-specific defaults
+package defaults
+
+import (
+	"time"
+
+	"github.com/gravitational/teleport"
+
+	"gopkg.in/square/go-jose.v2"
+)
+
+const (
+	// Namespace is default namespace
+	Namespace = "default"
+
+	// ServerKeepAliveTTL is a period between server keep-alives,
+	// when servers announce only presence without sending full data
+	ServerKeepAliveTTL = 60 * time.Second
+
+	// DefaultDialTimeout is a default TCP dial timeout we set for our
+	// connection attempts
+	DefaultDialTimeout = 30 * time.Second
+
+	// KeepAliveCountMax is the number of keep-alive messages that can be sent
+	// without receiving a response from the client before the client is
+	// disconnected. The max count mirrors ClientAliveCountMax of sshd.
+	KeepAliveCountMax = 3
+
+	// MaxCertDuration limits maximum duration of validity of issued cert
+	MaxCertDuration = 30 * time.Hour
+
+	// ApplicationTokenAlgorithm is the default algorithm used to sign
+	// application access tokens.
+	ApplicationTokenAlgorithm = jose.RS256
+
+	// KeepAliveInterval is interval at which Teleport will send keep-alive
+	// messages to the client. The default interval of 5 minutes (300 seconds) is
+	// set to help keep connections alive when using AWS NLBs (which have a default
+	// timeout of 350 seconds)
+	KeepAliveInterval = 5 * time.Minute
+)
+
+// EnhancedEvents returns the default list of enhanced events.
+func EnhancedEvents() []string {
+	return []string{
+		teleport.EnhancedRecordingCommand,
+		teleport.EnhancedRecordingNetwork,
```

Comment:
You're already moving constants over, I'd move these over as well and break the dependency on `github.com/gravitational/teleport`.

Reply:
`github.com/gravitational/teleport` is still being imported in several other spots in the api package, so I think at this point it'd be better served as a separate pr.

Reply:
Okay sounds good to me.